README.txt

```diff
@@ -0,0 +1,13 @@
+SDL_gpu_shadercross
+
+This is a library for translating shaders to different formats, intended for use with SDL's GPU API.
+It takes SPIRV or HLSL as the source and outputs DXBC, DXIL, SPIRV, MSL, or HLSL.
+
+This library can perform runtime translation and conveniently returns compiled SDL GPU shader objects from HLSL or SPIRV source.
+This library also provides a command line interface for offline translation of shaders.
+
+For SPIRV translation, this library depends on SPIRV-Cross: https://github.com/KhronosGroup/SPIRV-Cross
+For compiling to DXIL, dxcompiler.dll and dxil.dll are required (or your platform's equivalent).
```

Comment:
Do those also ship with Windows or else where do you get them?
Should the application using SDL_gpu_shadercross ship dxcompiler.dll and dxil.dll or should it expect the user to install some DirectX runtime?

Does it not need any additional libraries for macOS/Metal support?

(apart from this the README looks good to me :))

Reply:
> Do those also ship with Windows or else where do you get them? Should the application using SDL_gpu_shadercross ship dxcompiler.dll and dxil.dll or should it expect the user to install some DirectX runtime?

Added a note about this.
 
> Does it not need any additional libraries for macOS/Metal support?

Nope, just SPIRV-Cross.